R/derive_advs_params.R

```diff
@@ -530,13 +584,30 @@ compute_bsa <- function(height = height,
 #'   ),
 #'   get_unit_expr = extract_unit(PARAM)
 #' )
+#'
```

Comment:
Could you update the first example (for BMI and BSA) such that height is measured at every visit? The current example does not make much sense.

I would also change the order of the examples because measuring height only once is the more common case.

Reply:
@bundfussr updated order of examples, thanks!